### PR TITLE
Add -quiet option to pkcs7 for -print_certs

### LINKED ISSUE #17089: 'Clean' PKCS7 print_certs without the subject= & issuer= text lines
Please could you consider a 'clean' version of `print_certs` that **only** outputs the encoded certificate data, without adding the `subject=` & `issuer=` text lines before each encoded certificate block?

`openssl.exe pkcs7 -print_certs -in CAChain.p7b -out CAChain.cer`

The `print_certs` output for _OpenSSL 3.0.0 7 sep 2021_ is like this:

```
subject=DC = com, DC = domain, CN = InterCA
issuer=CN = RootCA
-----BEGIN CERTIFICATE-----
MIIGOzCCBCOgAwIBAgITZAAAAAhW1x9LPsw9uAABAAAACDANBgkqhkiG9w0BAQ0F
ADAWMRQwEgYDVQQDEwtNb211c1Jvb3RDQTAeFw0yMTAzMDUxMTM5MjhaFw00MTAz
~~
VMcF0teVDY1SgDyKhOEGn51jd3/9nVGLH+ck7Z7CzGAHwCRKAJjo2LcqzxC636Jc
HIiLRoD6DbEZYO1BvzFz
-----END CERTIFICATE-----

subject=CN = RootCA
issuer=CN = RootCA
-----BEGIN CERTIFICATE-----
MIIFMDCCAxigAwIBAgIQObxqmJ06R5ROVDx+9QHPFjANBgkqhkiG9w0BAQ0FADAW
MRQwEgYDVQQDEwtNb211c1Jvb3RDQTAgFw0yMTAzMDUxMTEzMjNaGA8yMDUwMTIy
~~
XjMus6H50cB0GZWzKJc87SgTi+4X484bjHyuMtFhsdisEukkYFxQ+6fsdX7qNU8A
/GZDXYQvybS7gxf2desxCEedxecMVrGH2isp+37/uMP+lndo
-----END CERTIFICATE-----
```

The  'clean' version would be simply:

```
-----BEGIN CERTIFICATE-----
MIIGOzCCBCOgAwIBAgITZAAAAAhW1x9LPsw9uAABAAAACDANBgkqhkiG9w0BAQ0F
ADAWMRQwEgYDVQQDEwtNb211c1Jvb3RDQTAeFw0yMTAzMDUxMTM5MjhaFw00MTAz
~~
VMcF0teVDY1SgDyKhOEGn51jd3/9nVGLH+ck7Z7CzGAHwCRKAJjo2LcqzxC636Jc
HIiLRoD6DbEZYO1BvzFz
-----END CERTIFICATE-----

-----BEGIN CERTIFICATE-----
MIIFMDCCAxigAwIBAgIQObxqmJ06R5ROVDx+9QHPFjANBgkqhkiG9w0BAQ0FADAW
MRQwEgYDVQQDEwtNb211c1Jvb3RDQTAgFw0yMTAzMDUxMTEzMjNaGA8yMDUwMTIy
~~
XjMus6H50cB0GZWzKJc87SgTi+4X484bjHyuMtFhsdisEukkYFxQ+6fsdX7qNU8A
/GZDXYQvybS7gxf2desxCEedxecMVrGH2isp+37/uMP+lndo
-----END CERTIFICATE-----
```

Thanks
M

### PR DESCRIPTION
**Pull request by rferrandop**
I opened this PR for the CLA be effective. (Closed the previous one #17693)

Fixes #17089 

Added a new `-quiet` option to show a "clean" output and be useful for future use with other flags.

##### Checklist

- [x] documentation is added or updated

### CODE REVIEW

File: doc/man1/openssl-pkcs7.pod.in [hunk 1]
```diff
@@ -63,6 +64,10 @@ Print out the full PKCS7 object.
 Prints out any certificates or CRLs contained in the file. They are
 preceded by their subject and issuer names in one line format.
 
+=item B<-quiet>
+
+When used with -print_certs, prints out a clean version.
```
Comment (t8m) [inline doc/man1/openssl-pkcs7.pod.in:69]: `...prints out just the PEM-encoded certificates without any other output.`

**Comment by rferrandop** [comment]
Hi again @t8m , I added the test

**Review by t8m — APPROVED** [review]

File: apps/pkcs7.c [hunk 2]
```diff
@@ -46,6 +46,8 @@ const OPTIONS pkcs7_options[] = {
     {"print", OPT_PRINT, '-', "Print out all fields of the PKCS7 structure"},
     {"print_certs", OPT_PRINT_CERTS, '-',
      "Print_certs  print any certs or crl in the input"},
+    {"quiet", OPT_QUIET, '-', 
+     "When used with -print_certs, it produces a clean output"},
```
Comment (paulidale) [inline apps/pkcs7.c:50]: s/clean/cleaner/ ??

**Review by paulidale — APPROVED** [review]
A fairly trivial grammar suggestion.  Good once fixed.

File: apps/pkcs7.c (hunk 2 above)
Comment (rferrandop) [reply apps/pkcs7.c:50]: Done

**Review by paulidale — APPROVED** [review]

**Comment by paulidale** [comment]
Merged, thanks for this.

**Pull request closed without merging**